deltacat/compute/compactor/compaction_session.py

```diff
@@ -68,9 +69,10 @@ def compact_partition(
         min_hash_bucket_chunk_size: int = 0,
         compacted_file_content_type: ContentType = ContentType.PARQUET,
         delete_prev_primary_key_index: bool = False,
-        read_round_completion: bool = False,
         pg_config: Optional[PlacementGroupConfig] = None,
         schema_on_read: Optional[pa.schema] = None,  # TODO (ricmiyam): Remove this and retrieve schema from storage API
+        rebase_source_partition_locator: Optional[PartitionLocator] = None,
+        rebase_source_partition_high_watermark: Optional[int] = None,
         deltacat_storage=unimplemented_deltacat_storage):
```

Comment:
Could we return the final result as the location to the file so the orchestrator could cache the results for easy discoverability?

Reply:
The location of the round completion file? Sure, that makes sense.

Reply:
yes